torchmetrics/classification/cohen_kappa.py

```diff
@@ -117,3 +117,10 @@ def compute(self) -> Tensor:
         Computes cohen kappa score
         """
         return _cohen_kappa_compute(self.confmat, self.weights)
+
+    @property
+    def is_differentiable(self):
+        """
+        The equation of cohen kappa score is differentiable
+        """
+        return True
```

Comment:
cohen kappa is not differentiable since the implementation is based on calculating the confusion matrix which in general is not differentiable

Reply:
Ohk, I will change it